frontend/www/js/omegaup/components/identity/Editv2.vue

```diff
@@ -103,69 +103,69 @@ import Autocomplete from '../Autocomplete.vue';
   },
 })
 export default class IdentityEdit extends Vue {
-  @Prop() identity!: omegaup.Identity;
+  @Prop({ default: null }) identity!: omegaup.Identity;
```

Comment:
acá ponerle un default de `null` a `omegaup.Identity` es ilegal porque eso no ademite nulo explícitamente:

```suggestion
  @Prop({ default: null }) identity!: omegaup.Identity | null;
```

Reply:
Listo